docs/relay/relay.md

```diff
@@ -11,23 +11,42 @@ Many factors impact how you connect to the remote host, and it can be tricky to
 
 * Using a [Relay server](https://docs.unity.com/relay/en/manual/relay-servers): the server is on the Internet with a public-facing IP that you and the host can reach. After each side binds with the Relay, they can establish a connection and send data to each other via the Relay server.
 
-Netcode doesn't offer tools to help you do a successful punch through a NAT. But, Unity Services provides a Relay service that can relay all Unity Transport based technology, like Netcode.
+Netcode doesn't offer tools to help you do a successful punch through a NAT. However, Unity Services provides a Relay service that can relay all Unity Transport based technology, like Netcode.
 
-# How does Relay work?
+# How to use Relay
 
-You need to install the Unity Relay SDK if you want to interact with the service. To install it, add `com.unity.services.relay` package to your project.
+The host of your game will need to request a Relay allocation and get a [join code](https://docs.unity.com/relay/en/manual/join-codes) for this allocation. The join code is a random string that your clients will use to join the correct server and allocation. The host and clients will then be able to communicate via the Relay server without disclosing their IP addresses and ports directly.
```

Comment:
This section is only a high level explanation of what is happening. The actual requesting of a join code and allocation is done in the code snippets below

Reply:
Oh sorry for not answering earlier, I just saw those questions. In case no one answered for you:
1- you are trying to establish a direct connection through stuff (firewall, router...) that prevent you from making a direct connection because they translate addresses. I have zero experience with that but I think you need some help from another server or so to kind of guess what address and port you need to contact the other client on. We do not help people with that, instead we recommend to use Relay ;-) 
2- Wether you have to go through firewalls/router that translate addresses (= use NAT) makes direct (P2P) connections harder
3- I would talk about "requesting an allocation on the relay server", which is done with the line of code 
`Allocation allocation = await RelayService.Instance.CreateAllocationAsync(maxConnections);`